lib/routes/autocentre/index.ts

```diff
@@ -0,0 +1,32 @@
+import { Data, Route } from '@/types';
+import parser from '@/utils/rss-parser';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '',
+    name: 'Автоцентр.ua',
+    maintainers: ['driversti'],
+    example: 'autocentre',
```

Comment:
example should start with `/`

Reply:
What feed category should I choose? I didn't find any category related to automotive. Can we add one?

Reply:
new-media

Reply:
Done